pkg/controller/volume/expand/expand_controller.go

```diff
@@ -260,18 +258,18 @@ func (expc *expandController) syncHandler(key string) error {
 		}
 
 		msg := fmt.Sprintf("CSI migration enabled for %s; waiting for external resizer to expand the pvc", inTreePluginName)
-		expc.recorder.Event(pvc, v1.EventTypeNormal, events.ExternalExpanding, msg)
+		expc.recorder.Eventf(pvc, nil, v1.EventTypeNormal, events.ExternalExpanding, "Expanding pvc", msg)
```

Comment:
According to https://github.com/kubernetes/kubernetes/blob/ea0764452222146c47ec826977f49d7001b0ea8c/staging/src/k8s.io/client-go/tools/events/interfaces.go#L37, the action should also be upper camel case. So something like "ExpandingPVC" would work.

Ditto everywhere